src/screens/ConfirmTransactionScreen.js

```diff
@@ -23,16 +23,16 @@ const ConfirmTransactionScreen = ({ navigation }) => {
     try {
       const tx = await Crypto.signDAITransaction(amount, toAddress);
       setProcessing(false);
-      let txData = {
-        toAddress: toAddress,
+      const txData = {
```

Comment:
@josephhaddad55 shouldn't this be a `let`? line 33 modifies `txData.toUsername`

Reply:
@msrijai When you're adding an array or object you're not re-assigning the constant. You're just adding the "list" or "property" in that case the constant `txtData` points to. 

[Check out this minimum example on repl](https://repl.it/@josephhaddad55/MobileStarkEnvironments)